### PR TITLE
WIP: Add qjs as a subproject ##build

### PR DESCRIPTION
**Pull request by satk0**
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [book](https://github.com/radareorg/radare2book) (optional)

**Description**

### CODE REVIEW

**Comment by trufae** [comment]
`libr/lang/meson.build:7:6: ERROR: Subproject exists but has no meson.build file.` you can add a meson.build in the qjs repo by adding it as a patch, the .wrap files can specify a directory for patches taht will be applied (iirc files copied too)

**Comment by satk0** [comment]
Oooh, I see, great idea 😃

**Comment by satk0** [comment]
hello windows devil, please be kind :pray:

**Comment by satk0** [comment]
idk how to upgrade muon, should I upgrade r2pm muon version or smth?

quick note: WIndows `preconfigure.bat` should also be updated to support newer frida meson version

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -390,7 +390,7 @@ jobs:
         arch: x64
     - name: Install dependencies
       run: |
-        pip install git+https://github.com/frida/meson.git@f7f25b19a8d71cebf8e2934733eb041eb6862eee --break-system-packages
+        pip install git+https://github.com/frida/meson.git@32af71f85f7eba55781e8404f2742a2c434f8c6f --break-system-packages
```
Comment (trufae) [inline .github/workflows/build.yml:393]: i think it will be better to just install standard meson, not frida's fork

**Pull request merged**